src/bridge/cockpitpackages.c

```diff
@@ -621,7 +621,7 @@ maybe_add_package (GHashTable *listing,
 
   if (bundle_checksum || paths)
     {
-      if (!package_walk_directory (bundle_checksum, own_checksum, paths, directory, NULL))
+      if (!package_walk_directory (directory, NULL, paths, bundle_checksum, own_checksum))
```

Comment:
Still wrong :rofl:

Reply:
OK, now I need a little more input which argument order you would actually prefer :grin:

Reply:
The problem is (and was) the order of the "own" and "bundle" checksum arguments.

Reply:
In what sense is that anything but arbitrary? I.e. why does one should æsthetically come first?

Do you agree to the root/directory/paths change, or do you have a different order in mind there?

Reply:
The order in the declaration is `own_checksum` then `bundle_checksum`.

```
static gboolean package_walk_directory (const gchar *root,
                                        const gchar *directory,
                                        GHashTable *paths,
                                        GChecksum *own_checksum,
                                        GChecksum *bundle_checksum);
```

But at the call site, `bundle_checksum` comes first.

```
       if (!package_walk_directory (directory, NULL, paths, bundle_checksum, own_checksum)) 
```

Reply:
For what it's worth, in `bridge.py` I just pass an array here :)

Reply:
*slaps forehead* of course :grin: Sorry for the misunderstanding. Fixed locally, will push after #17049 lands.